src/client/java/teammates/client/scripts/ListActiveInstructors.java

```diff
@@ -0,0 +1,43 @@
+package teammates.client.scripts;
+
+import java.time.Instant;
+import java.util.ArrayList;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.storage.entity.FeedbackSession;
+
+/**
+ * Script to generate emails of active instructors within a period.
+ */
+public class ListActiveInstructors extends RemoteApiClient {
+    @Override
+    protected void doOperation() {
+        //2020/01/01
+        long startTime = 1577808000000L;
+        //2021/02/01
+        long endTime = 1612108800000L;
```

Comment:
I'll have to update these manually. So, use a human editable format for the dates :-)